a.c

```diff
@@ -0,0 +1,35 @@
+#define OPENSSL_SUPPRESS_DEPRECATED
```

Comment:
I dont think you meant to include these 3 a.* files?

Reply:
sigh.  commit pushed to remove them.